README.md

```diff
@@ -14,11 +14,17 @@ The following procedure describes how to deploy the `ExternalDNS` Operator for A
    $ make image-build
    $ make image-push
    ```
-2. Run the following command to deploy the `ExternalDNS` Operator:
+2. Prepare the operator and operand namespaces:
+   ```
+   oc create ns external-dns-operator
```

Comment:
In case of `make deploy` deployment, the operator namespace is created in `config/`: https://github.com/openshift/external-dns-operator/blob/main/config/manager/manager.yaml#L6.

Reply:
Changed. thx